src/components/FHIRendpoint.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import { Icon } from '@iconify/react';
+import Advanced from './AdvancedOptions';
+
+function Endpoint() {
+  const [link, setLink] = useState('');
+  const [showAdvancedOptions, setShowAdvancedOptions] = useState(false);
+  const [buttonHover, setButtonHover] = useState(false);
+  const [buttonClick, setButtonClick] = useState(false);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+  };
+
+  const handleAdvancedOptionsClick = () => {
+    setShowAdvancedOptions((prev) => !prev);
+  };
+
+  const handleButtonHover = () => {
+    setButtonHover(true);
+  };
+
+  const handleButtonLeave = () => {
+    setButtonHover(false);
+  };
+
+  const handleButtonClick = () => {
+    setButtonClick(true);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="relative">
+        <label type="text" htmlFor="s1-2">
+          <div className="pt-3 pb-1">
+            <h1 className="font-sans font-bold text-md">FHIR Endpoint URL</h1>
+          </div>
+          <input
+            type="text"
+            value={link}
+            onChange={(e) => setLink(e.target.value)}
+            placeholder="https://api.examplePath.org/RTTD/open"
```

Comment:
Let's make the placeholder something more generic like `http://example.org/` or whatever, we don't want to make anyone think they need any specific queries with the slash extensions